src/plugins/intel_gpu/src/runtime/debug_configuration.cpp

```diff
@@ -136,6 +136,7 @@ static void print_help_messages() {
                               " For example fc:onednn gemm:onednn reduce:ocl do:cpu"
                               " For primitives fc, gemm, do, reduce, concat are supported. Separated by space.");
     message_list.emplace_back("OV_GPU_MaxKernelsPerBatch", "Maximum number of kernels in a batch during compiling kernels");
+    message_list.emplace_back("OV_GPU_ImplsCacheCapacity", "Size of kernel impl cache");
```

Comment:
Could you mention that the size is in terms of # of impls?

Reply:
oh good point. What about `The maximum number of entries in the kernel impl cache`?